src/api/createRequest.js

```diff
@@ -64,11 +64,12 @@ const parametersToString = (attributes, parameter = undefined) => {
  * @param {Object} filter - query filters
  * @param String include - query include
  * @param {Object} fields - query fields
+ * @param {Object} page - query page
  * @return String
  * Turns a list of parameters into a query string
  */
-const buildQuery = ({ filter = {}, include = '', fields = {} } = {}) => {
-  const queryString = parametersToString({ filter, include, fields })
+const buildQuery = ({ page = {}, filter = {}, include = '', fields = {} } = {}) => {
```

Comment:
Advisory:  wonder if it would be worth turning this into a type? I thought about this when I did it originally but needed to stop somewhere. get, find and create all use the build query method. Seems like it might be an idea to pass that type

Reply:
Yep. Job for a separate story, maybe something like "Add consistent typing across API requests builders?"